Source/Core/Core/HW/MMIO.cpp

```diff
@@ -304,6 +304,8 @@ void ReadHandler<T>::ResetMethod(ReadHandlingMethod<T>* method)
 
   struct FuncCreatorVisitor : public ReadHandlingMethodVisitor<T>
   {
+    virtual ~FuncCreatorVisitor() {}
```

Comment:
oh, ok. i was actually hoping just putting the definition in the header would do the same thing.